src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -91,10 +91,6 @@ internal static class FeatureOnOffOptions
             nameof(FeatureOnOffOptions), nameof(RefactoringVerification), defaultValue: false);
 
         [ExportOption]
-        public static readonly PerLanguageOption<bool> StreamingFindReferences = new PerLanguageOption<bool>(
-            nameof(FeatureOnOffOptions), nameof(StreamingFindReferences), defaultValue: true);
```

Comment:
Option was an 'escape hatch' in the early days of streaming find-refs in case we had an emergency and needed to go back to the old system.  The new system has proven itself at this point and can be the only impl for C# for VB at this point.